libr/asm/arch/mips/gnu/mips-dis.c

```diff
@@ -620,6 +620,16 @@ parse_mips_dis_option (const char *option, unsigned int len)
   val = option + (optionlen + 1);
   vallen = len - (optionlen + 1);
 
+	if (strncmp ("abi", option, optionlen) == 0
```

Comment:
This file seems to come from an external source and probably it was never modified. Are these changes really necessary here or can they be done somewhere else in our codebase?

Reply:
It's necessary. I set disassembler_options according to a->features, a->features comes from the analysis of ELF HEAD(I will add a function named Elf_(r_bin_elf_get_abi) to do that,